test.ts

```diff
@@ -4,9 +4,9 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { randomBytes } from 'node:crypto';
 import { decompress } from '@mongodb-js/zstd';
-import { TestCase } from './testCase.spec';
+import { TestCase } from './testCase.spec.js';
```

Comment:
Why is this change?

Reply:
Because it otherwise doesn't work TS config as is.

I don't know much TS, but this was necessary for it to type check with the tsconfig I added.

Check it out locally and see if you can get it to type check without the js 😀

Reply:
Apparently with `NodeNext`, that is indeed necessary.